features/step_definitions/topical_event_steps.rb

```diff
@@ -16,122 +11,12 @@
   create_topical_event_and_stub_in_content_store(name: name, summary: summary, description: description, start_date: 2.months.ago.to_date.to_s, end_date: Time.zone.today.to_s)
 end
 
-Then(/^I should see the topical event "([^"]*)" on the frontend is archived$/) do |topical_event_name|
-  topical_event = TopicalEvent.find_by!(name: topical_event_name)
-  visit topical_event_path(topical_event)
-  expect(page).to have_selector(".govuk-caption-xl.gem-c-title__context", text: "Archived")
-end
-
 Then(/^I should see the topical event "([^"]*)" in the admin interface$/) do |topical_event_name|
   topical_event = TopicalEvent.find_by!(name: topical_event_name)
   visit admin_topical_events_path(topical_event)
   expect(page).to have_selector(record_css_selector(topical_event))
 end
 
-Then(/^I should see the topical event "([^"]*)" on the frontend$/) do |topical_event_name|
```

Comment:
Could these step definition removals be squashed into the previous commit, where the steps are actually being deleted?